docs/tutorials/coredns.md

```diff
@@ -230,3 +230,35 @@ dnstools# dig @10.100.4.143 nginx.example.org +short
 10.0.2.15
 dnstools#
 ```
+
+## Customize prefix name in etcd
+`CoreDNS` provider supports an option to specify the prefix name.
+The prefix name is used as a part of key name in `etcd` for a DNS record.
+```
+$ external-dns --help
+...
+  --coredns-prefix=/skydns/    When using the CoreDNS provider, specify the prefix name
+...
+```
+
+To customize the prefix, you should change configurations in 2 places.
+
+For example, `/myprefix/` is used as the new prefix name instead of `/skydns/`.
+You should specify `--coredns-prefix` for `external-dns` command line to `/myprefix/`
+```
+      containers:
+      - name: external-dns
+        image: k8s.gcr.io/external-dns/external-dns:v0.7.3
```

Comment:
```suggestion
        image: k8s.gcr.io/external-dns/external-dns:v0.7.4
```

Reply:
@xunpan you should also update any other references to v0.7.3 to v0.7.4.

Reply:
I prefer to keep this PR for `document coredns-prefix feature`
I will upgrade related parts together with another PR. `external-dns` version should be one of them.

Reply:
@xunpan 👍